CrowdedMod/Patches/GenericPatches.cs

```diff
@@ -46,6 +46,17 @@ static class PlayerTabUpdateAvailableColorsPatch {
 				return false;
 			}
 		}
+		
+		[HarmonyPatch(typeof(ShipStatus), nameof(ShipStatus.GetSpawnLocation))]
+        	public static class ShipStatusGetSpawnLocationPatch
+		{
+		    public static bool Prefix(ShipStatus __instance, [HarmonyArgument(0)] ref int playerId, [HarmonyArgument(1)] ref int numPlayer, [HarmonyArgument(2)] bool initialSpawn)
+		    {
+			playerId %= 10;
+			if (numPlayer >= 10) numPlayer = 10;
+			return true;
```

Comment:
you don't need to use bool-prefix if it always returns true. void-prefix will do the same by [comment](https://github.com/CrowdedMods/CrowdedMod/pull/19#issuecomment-770851779)